diesel/src/upsert/mod.rs

```diff
@@ -1,4 +1,5 @@
 //! Types and functions related to PG's and Sqlite's `ON CONFLICT` clause
+//! If you use upsert, you need use PostgreSQL since version 9.0 or Sqlite3 since version 3.24.0.
```

Comment:
There should be an empty line between the "title" line and all other content.

Reply:
Fixed